tests/integration_tests/syscall.rs

```diff
@@ -1025,14 +1263,10 @@ fn sendto() {
         let dest_socket = UdpSocket::bind("127.0.0.1:0").expect("couldn't bind to address");
         let dest_port = dest_socket.local_addr().unwrap().port();
 
-        let server = thread::spawn(move || {
-            let mut buf = [0u8; EXPECTED.len()];
-            assert_eq!(
-                dest_socket.recv(&mut buf).expect("couldn't recv data"),
-                EXPECTED.len()
-            );
-            assert_eq!(buf, EXPECTED.as_bytes());
-        });
+        let server = thread::Builder::new()
+            .name(String::from("server"))
```

Comment:
personal preference:

```suggestion
            .name("server".into())
```